src/code_18F340.c

```diff
@@ -2,7 +2,20 @@
 
 INCLUDE_ASM(s32, "code_18F340", func_80260A60);
 
-INCLUDE_ASM(s32, "code_18F340", func_80260AD4);
+//INCLUDE_ASM(s32, "code_18F340", func_80260AD4);
+ApiStatus func_80260AD4(ScriptInstance* script, s32 isInitialCall) {
+    BattleStatus* battleStatus = &gBattleStatus;
+    Actor* playerActor = battleStatus->playerActor;
+    u8 var0 = playerActor->debuff - 3;
```

Comment:
similar story here for a u8 implying lots of ors. is this the same function again?